Visualizer/KIFEventVisualizer.m

```diff
@@ -5,40 +5,20 @@
 #import "KIFEventVisualizer.h"
 #import "KIFTouchVisualizerViewCoordinator.h"
 
-static BOOL __isKIFVisualizerCreated = NO;
-
-@interface KIFEventVisualizerWindow : UIWindow
-@end
-
-@implementation KIFEventVisualizerWindow
-
-- (UIWindowLevel)windowLevel
-{
-    return 100000099;
-}
-
-@end
-
-@interface KIFEventVisualizer()
+es@interface KIFEventVisualizer()
```

Comment:
?

Reply:
must have accidently typed something in there before commiting